home/import_helpers.py

```diff
@@ -157,3 +163,55 @@ def extract_errors(data: dict[str | int, Any] | list[str]) -> dict[str, str]:
         pass
 
     return error_message
+
+
+def fix_rows(rows: Generator[dict[str, str], None, None]) -> Iterator[dict[str, str]]:
+    """
+    Fix keys for all rows by lowercasing and removing whitespace
+    """
+    for row in rows:
+        yield fix_row(row)
+
+
+def fix_row(row: dict[str, str]) -> dict[str, str]:
+    return {k.lower().strip(): v.strip() if v else v for k, v in row.items()}
```

Comment:
The `if v else v` is confusing me here, I'm not sure it does anything? What are we trying to do here?

Reply:
I think this defends against a Nonetype exception. v.strip isn't possible if v is None.

Reply:
Yup. Removing it and running the tests, you get 

`AttributeError: 'NoneType' object has no attribute 'strip'` on a few tests.

Reply:
Yeah I'm thinking something more explicit would be better. Maybe we can separate it out into `normalise_key` and `normalise_value` functions, where we can have a
```
if v is None:
    return None
return v.strip()
```

But also, the typing for this function is incorrect, should the return then rather not be `dict[str, str | None]`?